media_driver/linux/gen12/ddi/media_sysinfo_g12.cpp

```diff
@@ -377,6 +377,12 @@ static bool adlsGt1Device4682 = DeviceInfoFactory<GfxDeviceInfo>::
 static bool adlsGt1Device4683 = DeviceInfoFactory<GfxDeviceInfo>::
     RegisterDevice(0x4683, &adlsGt1fInfo);
 
+static bool adlsGt1Device4688 = DeviceInfoFactory<GfxDeviceInfo>::
+    RegisterDevice(0x4688, &adlsGt1Info);
+
+static bool adlsGt1Device4689 = DeviceInfoFactory<GfxDeviceInfo>::
+    RegisterDevice(0x4689, &adlsGt1Info);
```

Comment:
0x4689 is removed in Mesa in commit https://gitlab.freedesktop.org/mesa/mesa/-/commit/3cc1efee6f9be11a2243ec4929ac8362197e26d6.

Reply:
This device ID is missing in the kernel list (at least in latest stable 6.0.8): https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/include/drm/i915_pciids.h?h=v6.0.8

Please, clarify where this device ID is coming from. Otherwise not LGTM.

Reply:
It came from the mesa driver, and as the comments above were mentioned, it has now been removed. I'll remove it.